packages/accessibility/src/behaviors/Button/buttonBehavior.ts

```diff
@@ -14,6 +14,8 @@ const buttonBehavior: Accessibility<ButtonBehaviorProps> = props => ({
     root: {
       role: props.as === 'button' ? undefined : 'button',
       tabIndex: props.as === 'button' ? undefined : 0,
+      disabled:
+        props.disabled || props.loading ? (props.as === 'button' ? true : undefined) : undefined,
```

Comment:
not sure about this, pls check with @kolaps33

Reply:
Look at the next line, it sets aria-disabled when `props.disabled || props.loading`.
I have just added a bit of consistency to have a native disabled attribute when it's a native button.